torch/onnx/_internal/_lazy_import.py

```diff
@@ -28,7 +28,7 @@ def __getattr__(self, attr: str) -> object:
 # NOTE: Add additional used imports here.
 if TYPE_CHECKING:
     import onnx
-    import onnx_ir  # type: ignore[import-untyped]
+    import onnx_ir  # type: ignore[import-untyped, import-not-found]
```

Comment:
Was this needed? It is installed in ci.

Reply:
We have different environments for different jobs, I guess the lint job doesn't install this?

Reply:
I'm not sure if this warrants the ignore - they can be multiple other optional dependencies that people don't need when they develop. But since they are installed in ci they are understood by mypy

Reply:
I'm not sure what you mean by "since they are installed in ci they are understood by mypy" - I have all the requirements.txt installed. I ran lintrunner locally. It complained. I fixed the complaint.

Reply:
Ok - that makes sense. On the other hand - if we're already saying that it's fine if this import doesn't have types available (ignore-untyped) then why does it matter if it's actually missing (for mypy) - the result will be the same, won't it?

Reply:
There was an issue with the package that we missed a py.type file to indicate that it was typed. I think the version used in ci was older at the time, but it should be fixed now!

Reply:
Ok - I'm convinced. If this PR fails to land I'll update it or I'll fix in a follow-up PR